release.py

```diff
@@ -516,7 +516,7 @@ def tag_repo(args):
         # tilde is not a valid character in git
         tag = '%s_%s' % (args.package_alias, args.version.replace('~', '-'))
         check_call(['git', 'tag', '-f', tag])
-        check_call(['git', 'push', '--tags'])
+        check_call(['git', 'push', 'origin', 'tag', tag])
```

Comment:
This assumes the remote is called `origin`. I used to name mine `upstream`, but I have sinced changed since a lot of our tools assume `origin`, but I'm not sure if it's a necessary assumption to make. Would this work without specifying the remote?